build/Dockerfile.nodeagent

```diff
@@ -2,7 +2,7 @@ FROM golang:1.13.1
 
 WORKDIR /go/src/github.com/IBM/ibm-block-csi-operator/
 COPY . .
-RUN CGO_ENABLED=1 GOOS=linux go build -o ibm-node-agent -gcflags all=-trimpath=${GOPATH} -asmflags all=-trimpath=${GOPATH} cmd/node/main.go
+RUN CGO_ENABLED=1 GOOS=linux go build -mod vendor -o ibm-node-agent -gcflags all=-trimpath=${GOPATH} -asmflags all=-trimpath=${GOPATH} cmd/node/main.go
```

Comment:
this is dead code, [no need to maintain it](https://github.com/IBM/ibm-block-csi-operator/pull/100#discussion_r501604925)

Reply:
oh, it's still being built. got you